xclim/sdba/adjustment.py

```diff
@@ -93,6 +93,21 @@ def _check_inputs(cls, *inputs, group):
                 f" this is not supported for {cls.__name__} adjustment."
             )
 
+        # Check multivariate dimensions
+        mvcrds = []
+        for inda in inputs:
+            for crd in inda.coords.values():
+                if crd.attrs.get("is_variables", False):
+                    mvcrds.append(crd)
+        if mvcrds and (
+            not all(mvcrds[0].equals(mv) for mv in mvcrds[1:])
```

Comment:
```len(set(mvcrds)) > 1``` ?

Reply:
That doesn't work because making a set needs hashable elements (that's how python tests for "equalness"). Here, I want the `DataArray`s to have equal dimensions and values and `DataArray`s are not hashable.